MonoGame.Framework.Content.Pipeline/Audio/AudioContent.cs

```diff
@@ -284,7 +284,10 @@ public void ConvertFormat(ConversionFormat formatType, ConversionQuality quality
 
                 // Loop start and length in number of samples. Defaults to entire sound
                 loopStart = 0;
-                loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
+                if (data != null && bitsPerSample > 0 && channelCount > 0)
+                    loopLength = data.Count / ((bitsPerSample / 8) * channelCount);
+                else
+                    loopLength = 0;
```

Comment:
Under what conditions can all this data...  the array, the sample bits, and the channel count.... be null?  Are these not required to correctly generate audio content?